receptor_satellite/run.py

```diff
@@ -64,10 +66,11 @@ async def run(self):
 
         await self.satellite_api.init_session()
         try:
+            self.queue.ack(self.playbook_run_id)
+            self.playbook = playbook_verifier.verify(self.playbook)
```

Comment:
I think we'll need to make this optional, however the setting should default to true.

Reply:
Do we? We rely on cloud to know if a client can or cannot process signed playbooks, I assumed this would be based solely on receptor-satellite version. If we make this optional, then cloud would have to know about version and value of the setting

Reply:
In case we want to disable it at the last minute, we can switch the commented and uncommented lines here https://github.com/project-receptor/receptor-satellite/blob/a96253b05daaddbc07de8cef1d50a1a6d391bbab/receptor_satellite/playbook_verifier_adapter.py